moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -145,9 +144,11 @@ class Robot
    * @brief execute a given trajectory
    * @param traj_name - name of trajectory to be executed (must be filled with
    * prior calls to "addTrajPoint".
-   * @return - true if desired trajectory was executed.
+   * \throws <std::runtime_error> (Execution failure)
```

Comment:
This method throws for multiple reasons.  Because it always throws the `std::runtime_error`, the calling code will not be able to handle these error cases separately.  Isn't this a problem?